pennylane/interfaces/autograd.py

```diff
@@ -419,7 +419,9 @@ def _get_jac_with_caching():
 
         jacs = []
         for t in tapes:
-            g_tapes, fn = gradient_fn(t, shots=device.shot_vector, **gradient_kwargs)
+            # assume that shot vector is always none on the new device interface for now
+            shot_vector = getattr(device, "shot_vector", None)

```

Comment:
So we do not have shot integration for the moment? We want to add it when the shots are on the tape?

Reply:
This was created before shots were on the tape.  Once shots are on the tape and the new device can handle them, we can go back and change this line.